coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 83.44
+    "covered_percent": 43.37
```

Comment:
I still don't understand why when I push that the coverage is different. On my local machine, the coverage is up to 88.52%. Like seriously what. I'll investigate the coverage difference tomorrow, but the same thing happened in a previous branch. When @jforgue pulled that test branch of mine, ran rspec on his machine, and pushed, coverage went up. So it's definitely my problem.